packages/react-conformance/src/defaultTests.tsx

```diff
@@ -146,6 +146,60 @@ export const defaultTests: TestObject = {
     });
   },
 
+  /** Component file handles classname prop */
+  'component-handles-classname': (componentInfo: ComponentDoc, testInfo: IsConformantOptions) => {
+    const { Component, wrapperComponent, helperComponents = [], requiredProps, customMount = mount } = testInfo;
+
+    const defaultEl = customMount(<Component {...requiredProps} />);
```

Comment:
I'll fix this but each test (`it()`) should be standalone, and mounting components should happen inside the test itself.